src/Refactoring-Core/RBAddParameterRefactoring.class.st

```diff
@@ -29,6 +29,15 @@ RBAddParameterRefactoring class >> addParameterToMethod: aSelector in: aClass ne
 		initializer: init
 ]
 
+{ #category : #'instance creation' }
+RBAddParameterRefactoring class >> addParameterToMethod: aSelector in: aClass newSelector: newSelector permutation: aColl initializer: init [ 
+	^self new addParameterToMethod: aSelector
```

Comment:
I do not understand why this method does not call the next one and also why it does not need a model?

Reply:
All refactorings have two ways to initialize an instance, one with a model, an another with a default model, the defaul model is defined just like that:

```
model

	^ model
		ifNil: [ model := ( RBClassModelFactory rbNamespace onEnvironment: self defaultEnvironment )
				name: self printString;
				yourself
			]
		ifNotNil: [ model ]
```